### PR TITLE
config: setup semantic release

### PR DESCRIPTION
**Pull request by sherwinski**
This PR sets up tooling needed to automatically deploy and release new versions of this library.

### CODE REVIEW

File: projects/ng-imgix/package.json [hunk 1]
```diff
@@ -8,5 +8,67 @@
...
+      ],
+
+      "@semantic-release/changelog",
+      "@semantic-release/npm",
+      [
+        "@semantic-release/git",
+        {
+          "assets": [
+            "src/**",
+            "../../dist/ng-imgix/**",
```
Comment (frederickfogerty) [inline projects/ng-imgix/package.json:54]: What happens when the actual npm package is published? It seems like the dist files should probably moved under the `ng-imgix/dist` directory?

**Review by frederickfogerty — APPROVED** [review]
Mostly looks good!

File: projects/ng-imgix/package.json (hunk 1 above)
Comment (sherwinski) [reply projects/ng-imgix/package.json]: This was surprisingly [more confusing](https://github.com/angular/angular-cli/issues/12979) to achieve than I originally expected 😡

**Comment by sherwinski** [comment]
Just an added note that this is what the `dist` directory looks like:

image

**Pull request merged**